lib/utils.js

```diff
@@ -150,7 +150,7 @@ function Changes() {
   var self = this;
   EventEmitter.call(this);
   this.isChrome = isChromeApp();
-  this.listeners = {};
+  this._listeners = {};
```

Comment:
I also assume this was a mistake, to overwrite the built-in `EventEmitter.prototype.listeners()` method.